src/pages/Schedules/Schedules.scss

```diff
@@ -0,0 +1,16 @@
+.schedules{
+
+    .subtitle{
+        margin-top: 10%;
+    }
+
+    .button{
```

Comment:
You don't use .class inside the main class in this case .schedules…

You pass a prop className to button component, and then apply it here.

so : 
`.schecdules{
  &__subtitle{},
  &__button {}
}`

Just follow the other components schemes:

Here is some documentation about this:
http://getbem.com/introduction/